src/Map/src/Bridge/Google/assets/src/map_controller.ts

```diff
@@ -194,6 +219,23 @@ export default class extends AbstractMapController<
                 infoWindow.open(this.map);
             });
 
+            if (definition.opened) {
+                const bounds = new google.maps.LatLngBounds();
+                element.getPath().forEach((point: google.maps.LatLng) => {
+                    bounds.extend(point);
+                });
+                infoWindow.setPosition(bounds.getCenter());
+                infoWindow.open({ map: this.map, anchor: element });
+            }
+        } else if (element instanceof google.maps.Polyline) {
+            element.addListener('click', (event: any) => {
+                if (definition.autoClose) {
+                    this.closeInfoWindowsExcept(infoWindow);
+                }
+                infoWindow.setPosition(event.latLng);
+                infoWindow.open(this.map);
+            });
```

Comment:
It looks like this logic is the same for Polygon, maybe we can merge the two blocks together?